<launch>
   <node name="goal_controller" pkg="lab01" type="goal_controller_node" output="screen">
      <param name="kv" value = "0.3" />
      <param name="kp" value = "0.5" />
      <param name="goal_range" value = "0.03" />
   </node>
</launch>
